recipes/pygit2/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pygit2" %}
+{% set version_major   = "0" %}
+{% set version_minor   = "25" %}
+{% set version_revison = "0" %}
+{% set version = ".".join([version_major, version_minor, version_revison]) %}
+{% set sha256 = "de0ed85fd840dfeb32bcaa94c643307551dc0d967c3714e49087e7edc0cdc571" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9    # [win and py27]
+    - vc10   # [win and py34]
+    - vc14   # [win and py35]
+
+requirements:
+  build:
+    - vc 9    # [win and py27]
+    - vc 10   # [win and py34]
+    - vc 14   # [win and py35]
+    - python
+    - setuptools
+    - cffi
+    - six
```

Comment:
It doesn't look like `six` is required for the build. Could we please drop it if that is the case?